cmd/queue/main.go

```diff
@@ -79,6 +79,7 @@ type config struct {
 	ServingReadinessProbe    string `split_words:"true" required:"true"`
 	EnableProfiling          bool   `split_words:"true"` // optional
 	EnableHTTP2AutoDetection bool   `split_words:"true"` // optional
+	EnablePodFreezer         bool   `split_words:"true"` // optional
```

Comment:
Do we want to name this "pod freezer"? IIRC, the proposal marketed the functionality as being useful for potentuial other cases as well. Essentially, what we want to do, is to add an endpoint that the queue-proxy calls when it's traffic drops to 0, right?

Should we, instead of straight up feature-gating this, add an option instead that specifies the location (or port or whatever) where the QP would make that call too? IF that setting is empty (the default) it does nothing. Like an `onNoMoreTraffic` hook or thelike. That would keep the general notion and would potentially make flows simpler as a whole. WDYT?

Reply:
That makes sense to me. We'd need to have some way to specify the endpoint in any case. If everyone's onboard with not putting a gate in front of it, I'll update the PR accordingly.